### PR TITLE
Add ability to set dashboard sharing options

### PR DESCRIPTION
**Pull request by rmoe**
Closes: #139

### CODE REVIEW

**Comment by tembleking** [comment]
Hi @rmoe , thank you for the contribution. Please, create a test in sysdig/resource_sysdig_monitor_dashboard_test.go that proves the behavior! 🙏

File: sysdig/resource_sysdig_monitor_dashboard.go [hunk 1]
```diff
@@ -53,6 +53,35 @@ func resourceSysdigMonitorDashboard() *schema.Resource {
...
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"member": {
+							Type:     schema.TypeSet,
+							MaxItems: 1,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"type": {
```
Comment (rmoe) [inline sysdig/resource_sysdig_monitor_dashboard.go:67]: Is there a list of all accepted types somewhere? Looking through the Python client I only see `TEAM` and `USER_TEAMS`.

File: sysdig/resource_sysdig_monitor_dashboard.go [hunk 2]
```diff
@@ -53,6 +53,35 @@ func resourceSysdigMonitorDashboard() *schema.Resource {
...
+										Required: true,
+									},
+									"id": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+								},
+							},
+						},
+						"role": {
```
Comment (rmoe) [inline sysdig/resource_sysdig_monitor_dashboard.go:78]: Similar to above, I only see `ROLE_RESOURCE_READ` and `ROLE_RESOURCE_EDIT` used in the Python client. If there is a definitive list of what's accepted I can update the docs accordingly.

File: sysdig/resource_sysdig_monitor_dashboard.go [hunk 3]
```diff
@@ -53,6 +53,35 @@ func resourceSysdigMonitorDashboard() *schema.Resource {
...
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"type": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"id": {
+										Type:     schema.TypeInt,
+										Required: true,
```
Comment (rmoe) [inline sysdig/resource_sysdig_monitor_dashboard.go:73]: I guess we can't make this required if we also support `USER_TEAMS` based on the API request the Python client makes.

**Comment by rmoe** [comment]
@tembleking tests have added, could you take another look at this?

**Comment by rmoe** [comment]
@tembleking @nestorsalceda @nkraemer-sysdig this PR and #147 have been open for a month now. What is it that I need to do to get these reviewed?

**Comment by tembleking** [comment]
Sorry for the delay @rmoe, I just updated your PR with a couple of fixes.
I will merge it ASAP, thank you for your patience.

**Pull request merged**